utilities/common.py

```diff
@@ -5,7 +5,9 @@
 
 def convert_dates_to_strings(data_dict: dict) -> dict:
     for key, value in data_dict.items():
-        if isinstance(value, datetime.date):
+        if key == "last_cached" and value is not None:
+            data_dict[key] = value.strftime("%Y-%m-%d %H:%M:%S")
+        elif isinstance(value, datetime.date):
```

Comment:
If we're going to have certain keys follow specific date formats, I'd rather the function be generalized rather than having a condition for a single key that is only relevant in one of the (many) places this function is called. I could see an optional secondary dictionary where the key is the data_dict key and the value is the desired strftime format for that. 

I won't block if we don't resolve this now, but if we don't, I would want a minor issue made so that we can address it later on.

Reply:
Gotcha! I didn't like doing it this way either haha. I figured maybe we could convert all `date` columns in the database to `datetime` as another solution, so that one `if` statement would work. But that seemed outside of scope

Reply:
I think that's an excellent idea, and probably a better choice -- I can't see any cases where we want date over datetime. If you make an issue for that (and for resolving this as well) I'll consider this resolved and will be happy to merge.